core/src/App.html

```diff
@@ -1,14 +1,17 @@
-<div id="app">
-    <div class="fd-page {hideNav? 'iframeContainerNoNav' : 'iframeContainer'}" use:init="context"></div>
-    <TopNav pathData={navigationPath} />
-    <LeftNav pathData={navigationPath} />
-</div>
+<Backdrop>
```

Comment:
I personally wouldn't add the Backdrop element outside the application container. And I believe we don't need to wrap any element inside the Backdrop. I would write this markup like that:
```
<div id="app">
    <Backdrop></Backdrop>
    <div class="fd-page {hideNav? 'iframeContainerNoNav' : 'iframeContainer'}" use:init="context"></div>
    <TopNav pathData={navigationPath} />
    <LeftNav pathData={navigationPath} />
</div>
```

Reply:
I think my approach is more natural and makes the code more referring to what we can actually see and do in the app. The backdrop wraps whole content and makes it unusable/unclickable, that's why I wrapped `content` in `backdrop`. I can change it if you really want, it's basically just code-understanding thing :)

Reply:
How about modifying your suggestion to
```
<div id="app">
    <Backdrop>
    <div class="fd-page {hideNav? 'iframeContainerNoNav' : 'iframeContainer'}" use:init="context"></div>
    <TopNav pathData={navigationPath} />
    <LeftNav pathData={navigationPath} />
</Backdrop>
</div>
```

?